app/routes/$orgSlug/settings/data-management/+data/DATA_DICTIONARY.md

```
@@ -0,0 +1,151 @@
+  "display_name": "Display Name",
+  "requested_at": "2024-01-15T10:00:00Z",
+  "reviewed_at": "2024-01-15T14:30:00Z",
+  "state": "APPROVED"
+}
+```
+
+- `state`: One of `APPROVED`, `CHANGES_REQUESTED`, `COMMENTED`, `DISMISSED`
+
+## Cycle Time Calculation
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)